recipes/liburing/all/conanfile.py

```diff
@@ -41,12 +48,21 @@ def configure(self):
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
 
-    def requirements(self):
-        self.requires("linux-headers-generic/5.13.9")
+    def _get_glibc_version(self):
```

Comment:
I think this will prevent us from crosscompiling this recipe.

Reply:
I don't understand purpose of croscompiling linux native library. But if you insist please check this comment [link](https://github.com/conan-io/conan-center-index/pull/7266#issuecomment-917643315). And if you still insist on enabling cross compilation for this particular library, a take a few days to investigate influence of kernel headers version on abi of this library

Reply:
This is the key to fix the underlying issue. It is not only about cross-compiling, it will prevent us to **use** this package with Conan on every platform other than Linux with `glibc>=2.27`. It is not just about building, it is about any Conan command (`conan info`, `conan lock`,...) that computes the graph of dependencies.